recipes/ccache-toolchain/activate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+export _OLD_PATH=${PATH}
```

Comment:
There is a problem with this approach. The order which activate/deactivate scripts is undefined.
So if you have other package that changes the PATH, ```_OLD_PATH``` can contain the wrong value.

For example:

```foo``` has an activate script that does:

```
export PATH="$FOO_DIR:$PATH"
```
and bar:


```
export _OLD_PATH="$PATH"
export PATH="$BAR_DIR:$PATH"
```

If foo's activate is executed first ```_OLD_PATH``` will contain ```$FOO_DIR```. Then if foo's deactivate is executed first, it will remove ```$FOO_DIR``` from ```PATH```, but on bar's deactivate it will revert ```PATH``` to ```_OLD_PATH```, which will still contain ```$FOO_DIR```

So I suggest doing this approach: https://github.com/gqmelo/gcc-recipes/blob/master/libgcc/deactivate-gcc-libs.sh
Also it would be good to copy the relevant tests as this is tricky to get it right: https://github.com/gqmelo/gcc-recipes/blob/master/libgcc/run_test.py